tests/acceptance/users_frontend.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Users Frontend
+  In order to manage users account in the web
+  As a user
+  I need to check user login and registration in joomla! CMS
+
+  Background:
+    Given I see the joomla! Home page
```

Comment:
Getting you Niels. but in the end I don't understand that we need background scenario or not. I think it is better to not use background scenario so far. I will add it later if I need it while writing test suits for it.